### PR TITLE
Add esa data parser to cubeviz

### PR DESCRIPTION
**Pull request by javerbukh**
### Description

This pull request is to address esa pipeline data not being loaded into cubeviz.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [x] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [ ] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [x] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Did the CI pass? If not, are the failures related?
- [x] Is a change log needed? If yes, is it added to `CHANGES.rst`?
- [x] Is a milestone set?
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

File: jdaviz/configs/cubeviz/plugins/parsers.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 __all__ = ['parse_data']
 
-EXT_TYPES = dict(flux=['flux', 'sci'],
+EXT_TYPES = dict(flux=['flux', 'sci', 'data'],
                  uncert=['ivar', 'err', 'var', 'uncert'],
                  mask=['mask', 'dq'])
```
Comment (pllim) [inline jdaviz/configs/cubeviz/plugins/parsers.py:16]: Does this need updating too to accomodate `('QUALITY', 'mask-viewer')`?
```suggestion
                 mask=['mask', 'dq', 'quality'])
```

**Review by pllim — COMMENTED** [review]
Since this is a hack on super secret data, it is ok to not test as long as Patrick okays this solution.

File: jdaviz/configs/cubeviz/plugins/parsers.py (hunk 1 above)
Comment (javerbukh) [reply jdaviz/configs/cubeviz/plugins/parsers.py:16]: Good call!

**Review by kecnry — APPROVED** [review]
Code looks reasonable enough to me (but haven't tested on the actual data locally)

**Review by PatrickOgle — APPROVED** [review]
Works as advertised, loading the data from FITS with a single command:
cubeviz.load_data("Cube.fits")

**Pull request merged**